packages/salesforcedx-vscode-lwc/resources/static/js-meta.xsd

```diff
@@ -0,0 +1,291 @@
+    </xs:sequence>
+  </xs:complexType>
+
+
+  <!-- objects SXD-->
+  <xs:complexType name="objects">
+    <xs:annotation>
+      <xs:documentation>
+        Limits the component to a set of one or more objects. If you don’t use this tag set, the component supports all supported objects. This tag set works only inside a parent targetConfig that’s configured for lightning__RecordPage.Specify the objects tag set only once inside a targetConfig set. Supports the object subtag.
+      </xs:documentation>
+    </xs:annotation>
+    <xs:sequence>
+      <xs:element type="xs:string" name="object" maxOccurs="unbounded" minOccurs="0">
+        <xs:annotation>
+          <xs:documentation>
+            Defines which objects the component is supported for. Use one object tag for each supported object. See the User Interface API Developer Guide for the list of supported objects.
+          </xs:documentation>
+        </xs:annotation>
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+
+  <!-- property SXD-->
+  <xs:complexType name="property">
+    <xs:annotation>
+      <xs:documentation>
+                Specifies a public property of a component that can be set in Lightning App Builder, App Manager, Lightning Flow Builder, or Experience Builder. The component author defines the property in the component’s JavaScript class using the @api decorator.
+          </xs:documentation>
+    </xs:annotation>
+    <xs:attribute type="xs:string" name="datasource" use="optional" />
+    <xs:attribute type="xs:string" name="default" use="optional" />
+    <xs:attribute type="xs:string" name="description" use="optional" />
+    <xs:attribute type="xs:string" name="label" use="optional" />
+    <xs:attribute type="xs:integer" name="max" use="optional" />
+    <xs:attribute type="xs:integer" name="min" use="optional" />
+    <xs:attribute type="xs:string" name="name" use="required" />
+    <xs:attribute type="xs:string" name="placeholder" use="optional" />
+    <xs:attribute type="xs:string" name="required" use="optional" />
+    <xs:attribute type="xs:string" name="role" use="optional" />
+    <xs:attribute type="xs:string" name="type" use="required" />
+  </xs:complexType>
+
+  <!-- supportedFormFactors SXD-->
+  <xs:complexType name="supportedFormFactors">
+    <xs:sequence>
+      <xs:element name="supportedFormFactor" maxOccurs="unbounded" minOccurs="1">
+        <xs:complexType>
+          <xs:annotation>
+            <xs:documentation>
+            Defines which form factor or device the component supports. Use one supportedFormFactor tag for each supported form factor.
+            </xs:documentation>
+          </xs:annotation>
+          <xs:attribute type="supportedFormFactortype" name="type" use="required" />
+        </xs:complexType>
+
+
+      </xs:element>
+    </xs:sequence>
+  </xs:complexType>
+
+  <xs:simpleType name="supportedFormFactortype">
+    <xs:restriction base="xs:string">
+      <xs:enumeration value="Large" />
```

Comment:
Can you add an xs:documentation for xs:enumeration tags for supportedFormFactortype?